Server/mods/deathmatch/logic/CVehicle.cpp

```diff
@@ -699,6 +700,7 @@ void CVehicle::SpawnAt(const CVector& vecPosition, const CVector& vecRotation)
 {
     SetHealth(GetRespawnHealth());
     SetIsBlown(false);
+    SetExplodeBroadcasted(true);

```

Comment:
Should this be `true` by default?

Reply:
The idea was to only set it to false at serverside when the vehicle will blow up.
But it shouldn't make any difference here because of SetIsBlown(false), could also remove it.
Only a safety thing to prevent any bugs in the future.